src/services/swap_service.ts

```diff
@@ -629,6 +636,15 @@ export class SwapService {
         };
     }
 
+    public async connectKafkaOrThrow(): Promise<void> {
+        if (!this._kafkaClient) {
+            throw Error('Kafka client was not passed in.');
+        }
+        const producer = this._kafkaClient.producer();
+        await producer.connect();
+        this.kafkaProducer = producer;
+    }
```

Comment:
Previously Kafka was optional, to run 0x API locally you didn't need to connect it. Is there any advantage to making it required, or could we keep it optional but still error out if it should connect but fails?